### PR TITLE
add translation

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
add translation

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |             |
| 🇨🇳 中文    |            |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/locales/zh.json [hunk 1]
```diff
@@ -1538,10 +1537,10 @@
     "jump server": "跳板机",
     "key only can be letters": "key只能为英文",
     "label tip": "标签提示",
-    "last {count} {unit}": "最近{count}{unit}",
     "last login time": "最近登录时间",
     "last month": "上个月",
     "last week": "上周",
+    "last {count} {unit}_0": "__NOT_TRANSLATED__",
```
Comment (daskyrk) [inline shell/app/locales/zh.json:1543]: Miss translation.

Comment (daskyrk) [reply shell/app/locales/zh.json:1543]: Key is not correct

Comment (wuhuiluo) [reply shell/app/locales/zh.json]: resolved

**Comment by daskyrk** [comment]
conflict

**Comment by wuhuiluo** [comment]
resolved

**Pull request merged**